src/Engine/ProtoCore/Utils/ArrayUtils.cs

```diff
@@ -471,5 +471,13 @@ public static bool IsEmpty(StackValue arrayPointer, RuntimeCore runtimeCore)
             var array = runtimeCore.Heap.ToHeapObject<DSArray>(arrayPointer);
             return array.Values.All(v => IsEmpty(v, runtimeCore));
         }
-   }
+
+        public static IEnumerable<T> GetCommonItems<T>(IEnumerable<T>[] lists)
```

Comment:
summary please on a public method

Reply:
I figured this method is self-explanatory but I'll add one nevertheless.